docker-compose.yml

```diff
@@ -376,7 +376,7 @@ services:
     command: sh -c ". /tmp/dev-tools/startup/startup.sh && scripts/start.sh"
     env_file: .env # used by the startup script
     volumes:
-      - ./creator-node/src:/usr/src/app/src
```

Comment:
Mounting everything causes problems
if there are no `node_modules` on host

Instead we should mount only directories with src
or use an empty mount for `node_modules`

Reply:
previously this did not mount test dependencies. i don't agree with only mounting src. why do we think node modules do not work?

Reply:
nvm i get it

Reply:
Do we also need to mount local storage?

Reply:
I don't think we should need to - I think removing it when downing & upping makes sense

Reply:
removal of local storage on up / down requires root permissions, however, volumes don't need this

Reply:
sorry I meant that it being cleared from the volume should be fine